documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -716,15 +722,6 @@ public readonly record struct GamepadState
 
 `Triggers` contains the two triggers on thsi gamepad. The values within this list range from 0 to 1: 0 being unpressed, and 1 being fully pressed.
 
-Note the use of the `DualReadOnlyList` type. This is basically just:
-```cs
-public readonly struct DualReadOnlyList<T> : IReadOnlyList<T>
-{
-    public readonly T Left;
-    public readonly T Right;
-}
-```
-
```

Comment:
Add this back please.

Reply:
it was already presented, why you want to i put it if you already explained the DualReadOnlyList<T> implementation because it would appear in the rest of the text?

Reply:
No idea what you’re saying, but I’m not intentionally removing an API declaration that is used throughout the rest of the proposal. It creates needless confusion.

Reply:
What I'm trying to say is you already put the implementation of the structure, a little above of where you said to i put back the info of where starts the part referencing the other proposal that I can't remember the name (sry) and i think make it two times is unnecessary

Reply:
Right, but this isn't an implementation detail, this is an API definition. It is normative and should be part of the specification.